### PR TITLE
Update sl container for sl7 to current (build-date=20190603)

### PR DESCRIPTION
**Pull request by sl-team**

### CODE REVIEW

**Comment by yosifkit** [comment]
Same comment as https://github.com/docker-library/official-images/pull/6009#issuecomment-498402094.

**Comment by jcpunk** [comment]
should be fixed now

File: library/sl [hunk 1]
```diff
@@ -8,7 +8,7 @@ Tags: 7, latest
 Architectures: amd64
 GitFetch: refs/heads/7
 Directory: sl7
-GitCommit: 340cfa6c6f8cb3f2b8427bd9271a57d30c3d7d4c
+GitCommit: 69fbf6ff9797b07eb95078df7f581124ac238879
```
Comment (yosifkit) [inline library/sl]:
```suggestion
GitCommit: 6f0afe07c9abcb8952c22ab76ed69d1fc757602e
```

Update to reflect the new commit id in https://github.com/scientificlinux/sl-docker/commits/7

**Comment by yosifkit** [comment]
<details>
<summary>Diff:</summary>

```diff
diff --git a/sl_latest/Dockerfile b/sl_latest/Dockerfile
index 8db9fb4..54217ca 100644
--- a/sl_latest/Dockerfile
+++ b/sl_latest/Dockerfile
@@ -2,7 +2,7 @@
 FROM scratch
 ADD sl-7-docker.tar.xz /
 
-LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20190501"
+LABEL name="SL7 Base Image" vendor="Scientific Linux" build-date="20190603"
 
 CMD ["/bin/bash"]
 
diff --git a/sl_latest/sl-7-docker.tar.xz b/sl_latest/sl-7-docker.tar.xz
index d39bd4f..5fe2d84 100644
Binary files a/sl_latest/sl-7-docker.tar.xz and b/sl_latest/sl-7-docker.tar.xz differ
```

</details>

Build test of #6010; 9d6f3a6a; `amd64` (`sl`):

```console

$ bashbrew build sl:7
Building bashbrew/cache:381a153710dfb993845bbb2b1323e220e85886254670524504984041827605c6 (sl:7)
Tagging sl:7
Tagging sl:latest

$ test/run.sh sl:7
testing sl:7
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed


$ bashbrew build sl:6
Using bashbrew/cache:d6fb8094533835a28208ac6f6daeb00285b32f58b1bc1dc8ffe8dbb48560cd86 (sl:6)
Tagging sl:6

$ test/run.sh sl:6
testing sl:6
	'utc' [1/4]...passed
	'cve-2014--shellshock' [2/4]...passed
	'no-hard-coded-passwords' [3/4]...warning: garbage password detected for 'root': 'locked'
passed
	'override-cmd' [4/4]...passed

```

**Pull request merged**